### ISSUE TITLE
add button "further commands" in wizard for setting up IR devices

### ISSUE THREAD

**Issue by stefaniehoechtl** [asterics/AsTeRICS-Ergo#6]
**Problem:**
When setting up the control of an IR device, the wizard proposes various remote commands. However, it is not clear how to add further commands.

**Solution:** 
Add the button „further commands“

**Issue resolved if:**
After recommending common commands, the user has the opportunity to add further commands.

**Comment by klues** [comment]
Haben wir uns hier schon überlegt, wie wir das genau machen wollen?
http://prntscr.com/gi9o3v
Bei (1), (2), oder (3) einen Button "weitere Befehle"?
Und dann wie beim schon existierenden Assistenten zum Hinzufügen weiterer Befehle? Zuerst Namen eingeben und dann lernen?

**Comment by stefaniehoechtl** [comment]
Genau, die User hätten den Button gerne neben Nr. 3. Ich würde sagen links daneben? Und anschließend soll man weitere Befehle hinzufügen können, indem man wie beim bereits existierenden Assistenten ("neuer Befehl") zunächst einen Namen eingibt und dann den Befehl lernt (durch Drücken der entsprechenden Taste auf der Fernbedienung).
